airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -375,7 +375,7 @@ def run_load(self,
         :type field_delimiter: string
         :param schema_update_options: Allows the schema of the desitination
             table to be updated as a side effect of the load job.
-        :type schema_update_options: list
```

Comment:
Nit: leave this as list, since you're just using the tuple as a trick to get around mutable default. Same below.